src/user/configure-programs-in-the-maintenance-app.md

```diff
@@ -1118,7 +1118,7 @@ by the data capture apps to identify this program stage.
 number of days to wait for starting the program stage.
 3. Enter repeatable program stage details.
     1. Specify if the program stage is **Repeatable** or not.
-    2. Select a **Period type**.
+    2. Select a **Period type** (when the period type on program stage works, it will mask the event date selection and replace it with a period picker).
```

Comment:
What is meant by "when the period type on program stage works"?

Reply:
@Bekkalizer perhaps you can clarify, as this is taken from your comment in the CoP :)